src/components/list/list.tsx

```diff
@@ -43,11 +47,18 @@ export class List {
     @Prop()
     public iconSize: IconSize = 'small';
 
+    /**
+     * Set to `true` if the list is open
+     */
+    @Prop()
+    public isOpen: boolean;
```

Comment:
Just adding a note here so I do not forget about it. We should investigate if we really need to add this, it's not very intuitive that a list has an `open` property. If we really need it, we might want to mark it as `internal` and add to the documentation that it's only used for the menu